packages/react/src/List/UnorderedList.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {createContext, type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+
+import styles from './List.module.css'
+
+export const ListContext = createContext({
```

Comment:
Same question, what's reason for using context here? [Related comment.](https://github.com/primer/brand/pull/145/files#r1053565117)